doc/01-installation-dev.rst

```diff
@@ -42,6 +42,27 @@ Install php 5.6 using command ``sudo apt-get install -y php5 php5-cli php5-intl
     echo "xdebug.var_display_max_depth=5" | sudo tee -a /etc/php5/mods-available/xdebug.ini > /dev/null
     sudo php5enmod common
 
+Install PHP 7.0
+~~~~~~~~~~~~~~~
+Enable http://dotdeb.org repository:
+
+  ::
+
+    wget https://www.dotdeb.org/dotdeb.gpg sudo apt-key add dotdeb.gpg
```

Comment:
I think that here you need separate to commands or add && between them. Otherwise `wget` will try to use `sudo apt-key add dotdeb.gpg` like additional arguments.